### ISSUE TITLE
Investigate Ads: Ad Free

### ISSUE THREAD

**Issue by VDuczekW** [guardian/dotcom-rendering#6379]
Discovery piece of work to review each container: If a page is ad-free, then some containers act differently

```[tasklist]
## Ad free blockers
- [x] https://github.com/guardian/dotcom-rendering/issues/7303
- [ ] https://github.com/guardian/dotcom-rendering/issues/7350
- [ ] https://github.com/guardian/dotcom-rendering/pull/7339
- [ ] https://github.com/guardian/dotcom-rendering/pull/7344
- [ ] https://github.com/guardian/dotcom-rendering/issues/7467
- [ ] https://github.com/guardian/dotcom-rendering/issues/7519
- [ ] https://github.com/guardian/frontend/pull/25999
```

```[tasklist]
## Questions
- [x] ~Are ads placed on DCR-rendered pages other than via `<AdSlot>`? If so, do we need to do anything to exclude these from ad free requests?~ There shouldn't be any other means of adding ads to the page that we would need to address in DCR.
- [x] ~Is the `isAdFreeUser` value that we're getting from Frontend accurate/canonical?~ In Frontend it's decided using the same method that we're using for Articles already.
- [x] ~Are there any ads that we should be showing even if `isAdFreeUser` is `true`? (e.g. the 'exclusion' ad slot in `HeaderAdSlot` https://github.com/guardian/dotcom-rendering/pull/6656 ? any others?)~ We've checked with comm-dev and we do not need to render ad slots if `isAdFreeUser` is `true` (see [Ashish's comment below](https://github.com/guardian/dotcom-rendering/issues/6379#issuecomment-1460349007)).
```

**Comment by VDuczekW** [comment]
Use this as teh epic to break down in tasks

**Comment by ashishpuliyel** [comment]
> Are there any ads that we should be showing even if `isAdFreeUser` is `true`? (e.g. the 'exclusion' ad slot in `HeaderAdSlot` https://github.com/guardian/dotcom-rendering/pull/6656 ? any others?)

Comm-Dev belive the behaviour of the exclusion slot is an oversight rather than by design. You should be ok to remove all ad-slots for ad-free users.

**Comment by bryophyta** [comment]
> > Are there any ads that we should be showing even if `isAdFreeUser` is `true`? (e.g. the 'exclusion' ad slot in `HeaderAdSlot` #6656 ? any others?)
> 
> Comm-Dev believe the behaviour of the exclusion slot is an oversight rather than by design. You should be ok to remove all ad-slots for ad-free users.

thanks @ashishpuliyel!

**Comment by rhiannareechaye** [comment]
hey @bryophyta would you mind putting a status update here, is there just one ticket left? [Don't render <AdSlot> in FrontLayout if renderAds is false.#7350](https://github.com/guardian/dotcom-rendering/issues/7350) ?

**Comment by bryophyta** [comment]
@rhiannareechaye I opened a PR for #7350 the other day and I've updated that issue to reflect that.

I've also just opened an issue to remove Labs containers (and cards?) from ad free fronts. There might be a little discovery to do there, but implementation will hopefully be straightforward.

That's all I'm aware of at the moment, aside from removing the `isAdFree` check from FaciaPicker (though that has to be done last, after everything else, so I've marked it as 'Do not merge').
